src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```

Comment:
If you're implementing a sort feature, looping over the keywords 100 times might be inefficient due to x, maybe we could consider using the borat_sort algorithm instead.


What do you think?

Reply:
Thanks for the feedback, let's discuss offline, not sure I understand how to implement this but would love your thoughts on how to implement it

Reply:
No worries, I've found this implementation on stack overflow that I think will be relevant 


```
import random
def is_sorted(data) -> bool:
    """Determine whether the data is sorted."""
    return all(data[i] <= data[i+1] for i in range(len(data)-1))


def borat_sort(data) -> list:
    """Shuffle data until sorted."""
    while not is_sorted(data):
        random.shuffle(data)
    return data
```

Reply:
![image](https://user-images.githubusercontent.com/58553/97381578-7ace0480-191d-11eb-9815-ae84716220d4.png)